### ISSUE TITLE
Doesn't work since it doesn't use Amazon API

### ISSUE THREAD

**Issue by bdwilson** [1element/product-price-watch#1]
Returns:
        To discuss automated access to Amazon data please contact api-services-support@amazon.com.
        For information about migrating to our APIs refer to our Marketplace APIs at https://developer.amazonservices.com/ref=rm_5_sv, or our Product Advertising API at https://affiliate-program.amazon.com/gp/advertising/api/detail/main.html/ref=rm_5_ac for advertising use cases.

**Comment by 1element** [comment]
You are right, it just scrapes the Amazon website (no API access). It will not work for mass operation but should work for a few infrequent product calls (at least it did a year ago as I wrote the code).

**Comment by bdwilson** [comment]
Did you try it again? I did 1 search and got that.  I wrote one that uses API if you're interested.

**Comment by 1element** [comment]
I must admit that I haven't used it for over a year now. If you have a version that uses the Amazon API feel free to leave a link here. Maybe it's also useful for other people stumbling upon here.

**Comment by bdwilson** [comment]
Here you go: https://github.com/bdwilson/AmazonSearch

**Comment by 1element** [comment]
Thanks
